proxies/dotnet/Controllers/User.cs

```diff
@@ -0,0 +1,47 @@
+using System.Net;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using ReadRawRequestBodyExample;
+
+namespace dotnet.Controllers;
+
+// users
+
+[ApiController]
+[Route("[controller]")]
+public class UserController : ControllerBase
+{
+
+    private readonly ILogger<UserController> _logger;
+    public static Dictionary<string, string> users = new Dictionary<string, string>();
+
+    public UserController(ILogger<UserController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateUser()
+    {
+        var body = await Request.GetRawBodyAsync().ConfigureAwait(false);
```

Comment:
couldn't you just use the User class from the SDK here as an argument to `CreateUser` rather than manually reading the body like this?

Reply:
I wasn't able to use the `DVCPopulatedUser` class unfortunately so I had to make its own model that could be used with model binding